src/libraries/WrappedSignatureLib.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+import {CalldataDecoder} from "./CalldataDecoder.sol";
+
+/// @title WrappedSignatureLib
+/// @notice A library for handling signatures with different wrapping schemes
+library WrappedSignatureLib {
+    using CalldataDecoder for bytes;
+
+    /// @notice Returns true if the signature is empty
+    /// @dev For use in the ERC-7739 sentinel value check
+    function isEmpty(bytes calldata data) internal pure returns (bool) {
+        return data.length == 0;
+    }
+
+    /// @notice Returns true if the signature does not contain any extra data (keyHash or hookData)
+    /// @dev Technically it will return false for WebAuthn signatures but they can't be verified without a keyHash
```

Comment:
This comments kinda confusing.. Can we maybe name this function like "isRawSignature" and just say returns true for standard ecdsa length signatures

and then @dev will also return true for standard p256 signatures however those MUST be wrapped with extra information in the verify sig flow

Reply:
Agree, thanks!